adit_radis_shared/common/tests/test_utils.py

```
@@ -0,0 +1,130 @@
+
+from adit_radis_shared.accounts.factories import UserFactory
+from adit_radis_shared.common.utils.async_utils import iter_over_async
+from adit_radis_shared.common.utils.auth_utils import is_logged_in_user
+from adit_radis_shared.common.utils.htmx_triggers import trigger_toast
+from adit_radis_shared.common.utils.mail import send_mail_to_admins, send_mail_to_user
+
+# --- mail helpers -----------------------------------------------------------
+
+
+def test_send_mail_to_admins_requires_some_content():
+    with pytest.raises(Exception):
```

Ruff (0.15.20)

[warning] Do not assert blind exception: `Exception`

(B017)